synapse/lib/stormhttp.py

```diff
@@ -102,7 +113,12 @@ async def _httpRequest(self, meth, url, headers=None, json=None, body=None, ssl_
         if proxyurl is not None:
             connector = aiohttp_socks.ProxyConnector.from_url(proxyurl)
 
-        async with aiohttp.ClientSession(connector=connector) as sess:
+        if timeout:
+            timeout = aiohttp.ClientTimeout(total=timeout)
+        else:
+            timeout = aiohttp.client.DEFAULT_TIMEOUT
```

Comment:
I think if we are using a default we should specify in the desc, or just make it the default val.

Reply:
set default timeout to 300s